services/payments/cancellationQueue.ts

```
@@ -0,0 +1,22 @@
+export class CancellationQueue {
+  private queue: string[] = [];
+  
```

ESLint

[error] Delete `··`

(prettier/prettier)